### DISCUSSION TITLE
Find my Geräte

### DISCUSSION THREAD

**Discussion by Citro12** [seemoo-lab/AirGuard#119] [Q&A] [answered]
Hallo,
was sind genau Find my Tracker außer Air Tags ?
Sind das Air Tags von Android Geräten zb Samsung etc . ?

What exactly are Find my Tracker other than Air Tags?
 Are these Air Tags from Android devices, e.g. Samsung etc.  ?

**Comment by Sn0wfreezeDev** [comment]
Hi, 
I will answer in English such that more people can understand. 
So Apple opened up their Bluetooth key finder protocol to other manufacturers. We summarize them under "other find my tracker", because they also work with the Find My App and Apple's Find My Network.
One of those is for example the Chipolo spot ONE. 

https://chipolo.net/en/pages/chipolo-one-spot

**Comment by Citro12** [comment]
Thanks :)

**Comment by Citro12** [comment]
one more question, normal iphone devices are recognized, not just mac books.  Would be interesting to know why?

**Comment by Myzel394** [comment, marked as answer]
> So Apple opened up their Bluetooth key finder protocol to other manufacturers.

@Sn0wfreezeDev could you post a link to that resource please?

**Reply by Myzel394** [reply]
@Sn0wfreezeDev I'm waiting for your response for almost a year now :(

**Reply by Sn0wfreezeDev** [reply]
This has been announced alongside the AirTag. Not sure what you are expecting. But there are plenty of other Find My Trackers available. https://developer.apple.com/find-my/

**Reply by Myzel394** [reply]
You were talking about a "Bluetooth key finder protocol" - do you know where to find this protocol?
